scripts/run.sh

```diff
@@ -67,7 +73,17 @@ fi
 # Create a unique container name
 CONTAINER_NAME="mcp-server-$(date +%s)-$RANDOM"
 
-# Run the container
+# Prepare arguments for MCP mode
+if [ "$MCP_MODE" = true ]; then
+  # Add the MCP mode flag for the CLI module
+  if [ -z "$ARGS" ]; then
+    ARGS="--mcp"
+  else
+    ARGS="--mcp $ARGS"
+  fi
+fi
+
```

Comment:
I think the server was modified to not need or use this flag anymore. Isn't this detected automatically by cli.py? Verify that and remove the unnecessary and unused "--mcp" flag.